src/routes/conversation/[id]/output/[sha256]/+server.ts

```diff
@@ -47,6 +47,9 @@ export const GET: RequestHandler = async ({ locals, params }) => {
 			"Content-Type": mime ?? "application/octet-stream",
 			"Content-Security-Policy":
 				"default-src 'none'; script-src 'none'; style-src 'none'; sandbox;",
+			"Content-Disposition": `attachment; filename="${sha256.slice(0, 8)}.${
+				mime?.split("/")[1] ?? "bin"
```

Comment:
I'm not sure this is accurate always, see https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types for extension/mime type matching

Reply:
fixed it using `mime-types`